recipe/meta.yaml

```diff
@@ -1,34 +1,41 @@
-{% set name = "typeguard" %}
```

Comment:
i've been leaning towards removing the `name` variable: while technically reused, it makes the URL _longer_ in many cases.